migrations/20230201519590_add_oauth_state_table.up.sql

```diff
@@ -0,0 +1,12 @@
+-- auth.oauth_state definition
+create table if not exists {{ index .Options "Namespace" }}.oauth_state(
+       id uuid primary key,
+       internal_auth_code varchar(255) null,
+       hashed_code_challenge varchar(255) unique null,
```

Comment:
Let's not add any uniqueness checks here. Hashed codes are generally unique, but also provided from the client side so there could be conflicts. They don't actually matter in this case anyway, so it's not really necessary.